.github/workflows/gh-ci.yaml

```
@@ -26,21 +26,28 @@ defaults:
     shell: bash -l {0}
 
 env:
-  MDANALYSIS_DEVELOP_VERSION: "2.3.0-dev0"
-  MDANALYSIS_LATEST_RELEASE: "2.2.0"
-  MDANALYSIS_PREVIOUS_RELEASE: "2.1.0"
+  MDANALYSIS_DEVELOP_VERSION: "2.5.0-dev0"
+  MDANALYSIS_LATEST_RELEASE: "2.4.3"
+  MDANALYSIS_PREVIOUS_RELEASE: "2.3.0"
 
 jobs:
   main-tests:
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
         os: [ubuntu-latest]
-        python-version: ["3.8", "3.9", "3.10"]
+        python-version: ["3.8", "3.9", "3.10", "3.11"]
```

```suggestion
        python-version: ["3.9", "3.10", "3.11"]
```

Probably can remove from test matrix?

---

looks to me like we should be using our new action 🙃